rocm.spec

```diff
@@ -104,6 +112,10 @@ rpm2cpio %{SOURCE29} | cpio -idmv
 rpm2cpio %{SOURCE30} | cpio -idmv
 rpm2cpio %{SOURCE31} | cpio -idmv
 
+%if "%{?SOURCE98:set}" == "set"
+cp %{SOURCE98} opt/rocm-%{realversion}/llvm/lib/clang/19/include/cuda_wrappers/bits
```

Comment:
@smuzaffar should this be `cp %{SOURCE98} %i/opt/rocm-%{realversion}/llvm/lib/clang/19/include/cuda_wrappers/bits`?

Reply:
Right now build_any_ib fails: https://cmssdt.cern.ch/SDT/jenkins-artifacts/build-any-ib/CMSSW_16_0_ROOT636_X_2025-11-06-1100/el8_amd64_gcc14/224610/external/rocm/7.0.2-26322fe51db58a9b402f3fa158a6bc12/log